setup.cfg

```diff
@@ -0,0 +1,50 @@
+[metadata]
+name = strava_py
+version = 0.0.1
```

Comment:
(It's possible to use things like https://github.com/pypa/setuptools_scm to make it automatically take the version number from Git tags, but let's start off simple.)